### DISCUSSION TITLE
Users actions in Grafana via approval request

### DISCUSSION THREAD

**Discussion by laa88rf** [grafana/grafana#64651] [General]
Hi.

In my work, a frequent problem is the lack of control of remove alerts, the lack of control over changes in requests in charts, and the change in response time to an alert.
Is it possible to create some kind of Merge Request? As far as I know, this is not the case in the current version.
Thank you.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82846. Feel free to continue the discussion around this there. Thank you!
